desktop/package.js

```diff
@@ -64,11 +73,14 @@ if (version) {
   startPack()
 } else {
   // use the same version as the currently-installed electron-prebuilt
-  exec('npm list | grep electron-prebuilt', (err, stdout, stderr) => {
-    if (err) {
-      DEFAULT_OPTS.version = '0.35.4'
-    } else {
-      DEFAULT_OPTS.version = stdout.split('@')[1].split(' ')[0]
+  console.log('Finding electron version')
+  exec('npm list --dev electron-prebuilt', (err, stdout, stderr) => {
```

Comment:
What's the issue?

On Thursday, January 14, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In desktop/package.js
> <https://github.com/keybase/client/pull/1713#discussion_r49801449>:
>
> > @@ -64,11 +73,14 @@ if (version) {
> >    startPack()
> >  } else {
> >    // use the same version as the currently-installed electron-prebuilt
> > -  exec('npm list | grep electron-prebuilt', (err, stdout, stderr) => {
> > -    if (err) {
> > -      DEFAULT_OPTS.version = '0.35.4'
> > -    } else {
> > -      DEFAULT_OPTS.version = stdout.split('@')[1].split(' ')[0]
> > +  console.log('Finding electron version')
> > +  exec('npm list --dev electron-prebuilt', (err, stdout, stderr) => {
>
> :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1713/files#r49801449>.
>